paper/paper.md

```diff
@@ -53,32 +53,38 @@ well-suited for the generation of benchmark-quality data, and to serve as a refe
 especially those relying on pseudopotentials.
```

Comment:
Trivial reformatting of the file, for max line length. Also fixed a word duplicate

Reply:
good find, thank you!